unittests/PhasarLLVM/Pointer/LLVMPointsToSetTest.cpp

```diff
@@ -58,6 +58,25 @@ TEST(LLVMPointsToSet, Global_01) {
   std::cout << '\n';
 }
 
+TEST(LLVMPointsToSet, SaveAndLoad) {
+  std::vector<std::string> testcases = {"pointers/basic_01_cpp.ll", "pointers/global_01_cpp.ll"};
```

Comment:
does this case test all the different cases that can happen during loading/storing?